README.md

```diff
@@ -219,3 +224,10 @@ pytest
 ```
 pipenv run black -l 120 .
 ```
+
+## Developing
+
+Install the package in editable mode so the changes take effect immediately.
+```
+pipenv install -e .
```

Comment:
This doesn't work for me. I still have to reinstall after each change in *.py files in `src/pg_to_evalscript`.

Reply:
This is very strange. For me it works correctly. After running `pipenv install -e .` `Pipfile` and `Pipfile.lock` now contain ` {editable = true, path = "."}` and if I change the files (and save changes), then the changes are visible when running tests.